src/registry_schemas/schemas/continuation_in.json

```diff
@@ -24,28 +25,86 @@
                 "foreignJurisdiction": {
                     "type": "object",
                     "required": [
-                        "name",
+                        "country",
                         "identifier",
                         "legalName",
-                        "incorporationDate"
+                        "incorporationDate",
+                        "affidavitFileKey"
```

Comment:
Affidavit is required only if foreign business is an AB or NS ULC.

Reply:
The UI will store a JSON property `isUlc` for its own purposes. I don't know if the API should use this to do API validation of affidavit. Thoughts?

Reply:
Forgot. will update it

Reply:
oh! the recent change in design. I think i need to introduce that field. Can they continue in as a CBEN if home jur type is ULC?

Reply:
I am not aware of any limitation but please ask Yui or Naveen.

Reply:
Which field needs to be added?

Reply:
The UI only tracks if it's ULC or Other.

Reply:
isULC flag is not required in schema